### ISSUE TITLE
text wrapping and alignement?

### ISSUE THREAD

**Issue by jaroslaw-weber** [PistonDevelopers/graphics#1081]
Is It possible to setup alignment and wrapping for Text?

**Comment by mitchmindtree** [comment]
@jaroslaw-weber I don't believe there are any tools for text alignment or wrapping in the `graphics` crate itself, however [conrod](https://github.com/PistonDevelopers/conrod) does provide this (and provides a backend for drawing to a piston `Graphics` instance if necessary - see the [all_piston_window](https://github.com/PistonDevelopers/conrod/blob/master/examples/all_piston_window.rs) example). Running the [text example](https://github.com/PistonDevelopers/conrod/blob/master/examples/text.rs#L129-L164) should give you an idea of how it works.
